src/electionguard_cli/__init__.py

```diff
@@ -111,8 +132,15 @@
     "e2e_input_retrieval_step",
     "e2e_inputs",
     "e2e_publish_step",
-    "e2e_submit_results",
     "election_builder_step",
+    "encrypt_ballot_inputs",
+    "encrypt_ballots",
```

Comment:
This is more of a note for later but we perhaps this particular module should only export the commands. `encrypt_ballots` feels like it could have easy collisions by accident with `encrypt_ballot` in `electionguard`